libs/libcommon/src/libcommon/queue.py

```diff
@@ -1010,6 +1060,19 @@ def get_jobs_total_by_type_and_status(self) -> JobsTotalByTypeAndStatus:
             )
         }
 
+    def get_jobs_count_by_worker_size(self) -> JobsCountByWorkerSize:
+        """Count the number of jobs by worker size.
+
+        Returns:
+            an object with the total of jobs by worker size.
+            Keys are worker_size, and values are the jobs count.
+        """
+        return {
+            WorkerSize.heavy.name: JobDocument.objects(difficulty__lte=100, difficulty__gt=70).count(),
+            WorkerSize.medium.name: JobDocument.objects(difficulty__lte=70, difficulty__gt=40).count(),
+            WorkerSize.light.name: JobDocument.objects(difficulty__lte=40, difficulty__gt=0).count(),
```

Comment:
just cor consistency

```suggestion
            WorkerSize.heavy.value: JobDocument.objects(difficulty__lte=100, difficulty__gt=70).count(),
            WorkerSize.medium.value: JobDocument.objects(difficulty__lte=70, difficulty__gt=40).count(),
            WorkerSize.light.value: JobDocument.objects(difficulty__lte=40, difficulty__gt=0).count(),
```